### DISCUSSION TITLE
为什么吸底模式不显示播放按钮？（如下图）English：Why does suction mode not show a play button? (Figure below)

### DISCUSSION THREAD

**Discussion by Super12138** [DIYgod/APlayer#685] [Q&A]
![image](https://user-images.githubusercontent.com/78189506/107115593-b0817280-68a8-11eb-8353-efd3304bc494.png)

**Comment by Ganxy03** [comment]
因为你就开了一个吸底模式 再开一个mini模式就ok了|Because you need to open the bottom mode and mini mode together
